mobile/native/src/api.rs

```diff
@@ -97,6 +97,21 @@ pub fn calculate_liquidation_price(
     ))
 }
 
+pub fn calculate_pnl(
+    opening_price: f64,
+    closing_price: f64,
+    quantity: f64,
+    leverage: f64,
+    direction: Direction,
+) -> SyncReturn<u64> {
+    // TODO: Handle the result and don't just return 0
```

Comment:
perhaps you could return `Option<u64>` here, that should be easiest to deal with in Dart